configuration_version.go

```diff
@@ -55,6 +55,18 @@ type ConfigurationVersions interface {
 
 	// Download a configuration version.  Only configuration versions in the uploaded state may be downloaded.
 	Download(ctx context.Context, cvID string) ([]byte, error)
+
+	// SoftDeleteBackingData soft deletes the configuration version's backing data
+	// **Note: This functionality is only available in Terraform Enterprise
+	SoftDeleteBackingData(ctx context.Context, svID string) error
+
+	// RestoreBackingData restores a soft deleted configuration version's backing data
+	// **Note**: This functionality is only available in Terraform Enterprise
+	RestoreBackingData(ctx context.Context, svID string) error
+
+	// PermanentlyDeleteBackingData permanently deletes a soft deleted configuration version's backing data
+	// **Note: This functionality is only available in Terraform Enterprise
```

Comment:
Could you catch the closing asterisks on the other two of these too? :) Thank you!

Reply:
Oops, not sure how I missed that one! I also changed the formatting to match the beta notes.